clients/@grouparoo/web/README.md

```diff
@@ -0,0 +1,121 @@
+# @grouparoo/client-web
+
+Grouparoo provides a javascript library for tracking events from browsers and sending them to your Grouparoo server.
+
+## Install
+
+You can include this library in a few ways:
+
+**Javascript / Typescript Module**
+
+For use in environments that will be compiled, ie: React or Webpack.
+
+```ts
+import { GrouparooWebClient } from "@grouparoo/client-web";
+const client = new GrouparooWebClient({
+  host: "https://grouparoo.my-company.com",
+  apiKey: "abc123zzz",
+});
+```
+
+**Script Tag**
+
+```html
+<html>
+  <head>
+    <script
+      type="application/javascript"
+      href="https://grouparoo.my-company.com/public/javascript/grouparooWebCleint.js"
+    />
+  </head>
+
+  <script>
+    const client = new GrouparooWebClient({
+      host: "https://grouparoo.my-company.com",
+      apiKey: "abc123zzz",
+    });
+  </script>
+</html>
+```
+
+To initialize a client, you'll need to provide:
```

Comment:
probably need an example of all of these jsut to be clear what's expected.

Reply:
I'll clarify that's what's happening in the above examples